.azure-pipelines/js.yml

```diff
@@ -0,0 +1,149 @@
+trigger:
+  - master
+
+variables:
+  NodeVersion: "10.x"
+  skipComponentGovernanceDetection: true
+
+jobs:
+  - job: "Build"
+
+    pool:
+      vmImage: "ubuntu-16.04"
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "$(NodeVersion)"
+        displayName: "Install Node.js $(NodeVersion)"
+
+      - script: |
+          cd src/ts/eslint-plugin-azure-sdk
+        displayName: "Reach directory"
+
+      - script: |
+          npm install -g npm@6.9.0
+        displayName: "Install npm version 6.9.0"
+
+      - script: |
+          npm install
+        displayName: "Install dependencies"
+
+      - script: |
+          npm run build
+        displayName: "Build library"
+
+      - script: |
+          npm pack
+        displayName: "Pack libraries"
+
+      - task: CopyFiles@2
+        inputs:
+          contents: "src/ts/eslint-plugin-azure-sdk/*.tgz"
+          targetFolder: $(Build.ArtifactStagingDirectory)
+          flattenFolders: true
+        displayName: "Copy packages"
+
+      - task: PublishBuildArtifacts@1
+        condition: succeededOrFailed()
+        displayName: "Publish artifacts"
+        inputs:
+          artifactName: packages
+
+  - job: "Analyze"
+
+    pool:
+      vmImage: "ubuntu-16.04"
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "$(NodeVersion)"
+        displayName: "Install Node.js $(NodeVersion)"
+
+      - script: |
+          cd src/ts/eslint-plugin-azure-sdk
+        displayName: "Reach directory"
+
+      - script: |
+          npm install -g npm@6.9.0
+        displayName: "Install npm version 6.9.0"
+
+      - script: |
+          npm install
+        displayName: "Install dependencies"
+
+      - script: |
+          npm run lint
+        displayName: "Run lint"
+
+      - script: |
+          npm audit
+        condition: and(succeeded(), eq(variables['RunNpmAudit'], 'true'))
+        displayName: "Audit packages"
+
+  - job: "Test"
+
+    strategy:
```

Comment:
@arpanlaha Let's remove this matrix and test only on one vm (ubuntu) and one node version specified above in the variables `NodeVersion: "10.x"` 
For reference: https://github.com/Azure/azure-sdk-tools/blob/master/.azure-pipelines/dotnet.yml#L38

Reply:
done!